api/src/main/java/com/datatorrent/api/SSLConfig.java

```diff
@@ -0,0 +1,82 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.datatorrent.api;
+
+import java.io.Serializable;
+
+/**
+ *
+ */
+public class SSLConfig implements Serializable
+{
+
+  private static final long serialVersionUID = -3491488868092056793L;
+  private String keyStorePath;
+  private String keyStorePassword;
+  private String keyStoreKeyPassword;
+  private String nodeLocalConfigPath;
```

Comment:
I can rename to configPath but I wanted to emphasize this is config file already supposed to be present as a local file on the target node (app master).